user-management/roles/add-zos-user/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+galaxy_info:
+  author: Blake Becker
+  description: Ansible z/OS Core Developer
+  company: IBM
+
+  license: Apache-2.0
+
+  min_ansible_version: 2.9
+
+  galaxy_tags: []
+
+
+dependencies: []
```

Comment:
For meta information, you might want to follow this template, so that we have some consistency across the sample playbooks. 

https://github.com/IBM/z_ansible_collections_samples/blob/master/copy-sort-fetch/roles/copy-local-data/meta/main.yml

Or we can also follow this template, as long as we're consistent.

Reply:
Updated to match your meta format